Makefile

```diff
@@ -30,13 +30,25 @@ debug-dockerize:
 	$(MAKE) -C http-gateway			debug-dockerize
 	$(MAKE) -C topic-controller		debug-dockerize
 
-kubectl-apply:
+dev-setup: clean dockerize
```

Comment:
I wouldn't put clean dockerize as dependencies.

dev-setup and teardown need to be declared as PHONY